### ISSUE TITLE
Issue with code provided

### ISSUE THREAD

**Issue by mbalsamo2** [learn-co-students/integrating-react-and-redux-codealong-v-000#273]
There is still the same issue that was raised months previously. A solution is provided within the responses to the original issue raised. Please fix so the code along will work once we get to the end of the lesson!

**Comment by IngridWong0715** [comment]
I totally agree. It seems like the issue was first raised almost a year ago, but still hasn't been fixed. It's really confusing and I think all students would really appreciate having a codealong that actually works.

**Comment by ylhoony** [comment]
agree, this codealong does not function at the end.

**Comment by jacobsilver2** [comment]
I'm having the same problem.
